manifests/params.pp

```diff
@@ -29,9 +29,21 @@
   if $::operatingsystem == 'Ubuntu' {
     if versioncmp($::lsbdistrelease, '8.04') < 1 {
       $init_style = 'debian'
-    } else {
+    } elsif versioncmp($::operatingsystemrelease, '15.04') < 0 {
```

Comment:
I can't be authoritative on that comment.  I don't know Ubuntu that far back to know if puppet supported $::operatingsystemrelease fact back then, I've heard that there was no $::operatingsystemmajrelease in older versions of some linux distros.  So I figured i'd let it stay, just in case as i'm not the pro on old linux systems.